virt/node_maintenance/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,13 @@
+[id="virt-using-skip-node"]
+= Using skip node annotation
```

Comment:
Suggestion: I wonder if a better assembly title would be something like "Preventing node reconciliation," because unless the user already knows what skip node annotation is, they might not know why they should read this.